mod/blog/elgg-plugin.php

```diff
@@ -14,6 +14,57 @@
 		'blog/auto_save_revision' => [],
 		'blog/delete' => [],
 	],
+	'routes' => [
```

Comment:
We can add an opiton to make the route public and automatically register it for walled garden.